backend/config/application.rb

```diff
@@ -13,6 +13,11 @@ module BrokenTableServer
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.1
+    
+    initializer(:remove_action_mailbox_and_activestorage_routes, after: :add_routing_paths) { |app|
```

Comment:
What do all these changes do?

Reply:
Rails automatically adds a bunch of unnecessary routes. This change is to remove the default routes that aren't appropriate.